### ISSUE TITLE
make blacklist work with full names if that is the valuetype selected

### ISSUE THREAD

**Issue by scottwleonard** [benkeen/react-country-region-selector#40]
I am using the selector to populate a list of countries. the user should be able to pick each country once and only once. I was going to use the blacklist to facilitate this, but ran into a problem;

right now, if I want to use full names in my data store, but want to blacklist selected values I need to somehow retrieve the short name based on the full name from the raw data. 

It seems logical that if I am using the full names, the blacklist should operate from the full name as well.

**Comment by benkeen** [comment]
Agreed, this is very reasonable. Hmph, how to do it without introducing a breaking change...
